### PR TITLE
Using below op aggfn instead of above op aggfn in MergeTwoAggRule

### PR DESCRIPTION
**Pull request by ikaruga4600**
ref #1230

### CODE REVIEW

**Comment by HangyuanLiu** [comment]
Please Add UT to cover this case

File: fe/fe-core/src/main/java/com/starrocks/sql/optimizer/rule/transformation/MergeTwoAggRule.java [hunk 1]
```diff
@@ -112,7 +112,9 @@ public boolean check(OptExpression input, OptimizerContext context) {
 
             CallOperator newFn;
             if (aggCallMapBelow.containsKey(ref)) {
-                newFn = new CallOperator(fn.getFnName(), fn.getType(), aggCallMapBelow.get(ref).getChildren(),
+                CallOperator belowOp = aggCallMapBelow.get(ref);
+                fn.getFunction().setArgTypes(belowOp.getFunction().getArgs());
+                newFn = new CallOperator(fn.getFnName(), fn.getType(), belowOp.getChildren(),
```
Comment (Seaven) [inline fe/fe-core/src/main/java/com/starrocks/sql/optimizer/rule/transformation/MergeTwoAggRule.java:117]:
```suggestion
                CallOperator belowOp = aggCallMapBelow.get(ref);
                newFn = new CallOperator(fn.getFnName(), fn.getType(), belowOp.getChildren(), belowOp.getFunction(), fn.isDistinct());
```

Comment (Seaven) [reply fe/fe-core/src/main/java/com/starrocks/sql/optimizer/rule/transformation/MergeTwoAggRule.java:117]: Nice work, but suggest don't modify the args type of function(it's a fixed signature, modify it maybe cause BE crash), you can use below aggregate function direct for the bug.

**Comment by Seaven** [comment]
And you can add UT by `explain verbose`, it can print args type

**Review by Seaven — APPROVED** [review]

**Review by Youngwb — APPROVED** [review]

**Pull request merged**